utils/scripts/install-centos8.sh

```diff
@@ -68,3 +67,15 @@ dnf --nodocs install \
     valgrind-devel \
     which \
     yasm
+
+# For fedora, java-11 is installed along with maven if we install maven from
+# repo. But we need java-8 (1.8). The 'devel' package also needs to be
+# installed specifically.
+
+if [ -e /etc/fedora-release ]; then
+        dnf install java-1.8.0-openjdk-devel maven-openjdk8
+else
+        dnf install maven
+fi
+
+dnf clean all
```

Comment:
This line isn't needed now, the dockerfile will do this after calling the scipt.

Reply:
removed